### PR TITLE
Add Integration test in python example with docs

### PR DESCRIPTION
**Pull request by wuqunfei**

### CODE REVIEW

File: testing-integration-py/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+##  Running the tests
+```
+pip install -r requirements.txt
+python -m unittest test_s3_it.py
+```
+
+## Test Life Cycle
+ - create a stack for all the use cases, to get all the output from stack
+ - assert any output value you defined in advance.
+ - In the end, don't forget destroy the stack.
```
Comment (komalali) [inline testing-integration-py/README.md:40]:
```suggestion
 - Create a stack and export the desired outputs.
 - Validate any output values you defined in advance.
 - In the end, don't forget to destroy the stack. 
```

File: testing-integration-py/test_s3_it.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+        cls.stack = auto.create_or_select_stack(stack_name=cls.STACK_NAME, work_dir=cls.WORK_DIR)
+        cls.stack.set_config("aws:region", auto.ConfigValue(value=cls.REGION_NAME))
+        cls.stack.up(on_output=print)
+        cls.outputs = cls.stack.outputs()
+        cls.s3 = boto3.resource('s3')
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        cls.stack.destroy(on_output=print)
```
Comment (komalali) [inline testing-integration-py/test_s3_it.py:27]: You might want to remove the stack completely at the end.

```suggestion
        cls.stack.destroy(on_output=print)
        cls.stack.workspace.remove_stack(cls.STACK_NAME)
```

File: testing-integration-py/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+```
+from pulumi.x import automation as auto
+
+class TestS3(unittest.TestCase):  
+  
+    @classmethod  
+    def setUpClass(cls) -> None:  
+      ....
+        cls.stack = auto.create_or_select_stack(stack_name=cls.STACK_NAME, work_dir=cls.WORK_DIR) 
+        cls.outputs = cls.stack.outputs()
```
Comment (komalali) [inline testing-integration-py/README.md:50]: Might be helpful to specify that an `up` is necessary before pulling outputs.
```suggestion
		cls.stack.up(output=print)
        cls.outputs = cls.stack.outputs() 
```

File: testing-integration-py/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    @classmethod  
+    def setUpClass(cls) -> None:  
+      ....
+        cls.stack = auto.create_or_select_stack(stack_name=cls.STACK_NAME, work_dir=cls.WORK_DIR) 
+        cls.outputs = cls.stack.outputs() 
+       ...
+          
+    @classmethod  
+    def tearDownClass(cls) -> None:  
+        cls.stack.destroy(on_output=print)
```
Comment (komalali) [inline testing-integration-py/README.md:55]: You might want to remove the stack completely at the end.

```suggestion
        cls.stack.destroy(on_output=print)
        cls.stack.workspace.remove_stack(cls.STACK_NAME)  
```

**Review by komalali — CHANGES_REQUESTED** [review]
Whoops, accidentally hit approve. Once you update the changes I recommended I'd be happy to approve it.

**Review by wuqunfei — COMMENTED** [review]
thanks, you made it perfect.

**Review by komalali — APPROVED** [review]

**Pull request merged**